packages/bierzo-wallet/src/routes/welcome/components/ExtensionInteraction.tsx

```diff
@@ -0,0 +1,27 @@
+import { JsRpcRequest } from '@iov/core';
+import React from 'react';
+
+export class ExtensionInteraction extends React.Component {
+  componentDidMount(): void {
```

Comment:
This is old school, we do not follow this pattern anymore, but congratulations you figured out! Do not worry, we'll refactor soon (we use react hooks now).

Reply:
okay, thanks, done. Using `React.useEffect` now.